catalog/glue.go

```diff
@@ -30,22 +30,40 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/glue/types"
 )
 
-const glueTypeIceberg = "ICEBERG"
+const (
+	glueTypeIceberg      = "ICEBERG"
+	databaseTypePropsKey = "database_type"
+	tableTypePropsKey    = "table_type"
+	descriptionPropsKey  = "Description"
+
+	// Database location.
+	locationPropsKey = "Location"
+
+	// Table metadata location pointer.
+	metadataLocationPropsKey = "metadata_location"
```

Comment:
is there documentation that these came from? Can we link to the source documentation that these came from?

Reply:
I could not find the official spec but followed the same convention used by the PyIceberg [here](https://github.com/apache/iceberg-python/blob/main/pyiceberg/catalog/__init__.py#L88). Let me know if this works.

Reply:
I think you also need to set `previous_metadata_location` when committing: See https://github.com/apache/iceberg-python/blob/main/pyiceberg/catalog/glue.py#L142-L145

Reply:
Added the documentation to the pyiceberg.

@nastra I will add `previous_metadata_location` in the next PR with the remaining ops (e.g. register_table) as it is used in them.